moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -72,15 +73,14 @@ ament_target_dependencies(${SERVO_PARAM_LIB_NAME} ${THIS_PACKAGE_INCLUDE_DEPENDS
 
 # This library provides a C++ interface for sending realtime twist or joint commands to a robot
 add_library(${SERVO_LIB_NAME} SHARED
-  # These files are used to produce differential motion
   src/collision_check.cpp
   src/enforce_limits.cpp
   src/servo.cpp
   src/servo_calcs.cpp
 )
 set_target_properties(${SERVO_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 ament_target_dependencies(${SERVO_LIB_NAME} ${THIS_PACKAGE_INCLUDE_DEPENDS})
-target_link_libraries(${SERVO_LIB_NAME} ${SERVO_PARAM_LIB_NAME})
+target_link_libraries(${SERVO_LIB_NAME} ${SERVO_PARAM_LIB_NAME} )
```

Comment:
It looks like it is failing the formatting check for cmake in CI for this extra space here.